go/test/endtoend/cluster/vtgate_process.go

```diff
@@ -80,7 +80,7 @@ func (vtgate *VtgateProcess) Setup() (err error) {
 		"-gateway_implementation", vtgate.GatewayImplementation,
 		"-service_map", vtgate.ServiceMap,
 		"-mysql_auth_server_impl", vtgate.MySQLAuthServerImpl,
-		"-pid_file", vtgate.PidFile,
+		//"-pid_file", vtgate.PidFile,
```

Comment:
what's the problem with the pid_file?

Reply:
I did this during some trial to fix the last 2 testcases (multisplitdiff, multi). I will revert once all things are settled.